### PR TITLE
docs: Clarify in docs and readme that Rook focuses on Ceph

### PR DESCRIPTION
**Pull request by travisn**
**Description of your changes:**
Rook has for quite some time only been focused on the Ceph operator. Maintainers have only been focused on Ceph, and the community has likewise been focused on Ceph. Now the docs are updated to clarify that Ceph storage is the focus and there are no plans for the project to support other storage providers.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,20 +11,28 @@
...
 
+## Ceph Provider
+
+The status of the Ceph storage provider is **Stable**.
+Features and improvements will be planned for many future versions. Upgrading between versions will be provided
+to ensure backward compatibility between releases.
+
+| Description                                                                                                                                                    | API Group       | Status |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | ------ |
+| [Ceph](https://ceph.com/) is a distributed storage system that provides file, block and object storage and is deployed in large scale production clusters. | ceph.rook.io/v1 | Stable |
```
Comment (galexrt) [inline README.md:31]: How about removing the table completely and moving the information out of it?

**Review by satoru-takeuchi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (travisn) [reply README.md]: Removed this section and integrated a paragraph on the Stable storage with the main section

**Review by galexrt — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -11,20 +11,20 @@
 
 # What is Rook?
 
-Rook is an open source **cloud-native storage orchestrator** for Kubernetes, providing the platform, framework, and support for a diverse set of storage solutions to natively integrate with cloud-native environments.
+Rook is an open source **cloud-native storage orchestrator** for Kubernetes, providing the platform, framework, and support for Ceph storage to natively integrate with Kubernetes.
 
-Rook turns storage software into self-managing, self-scaling, and self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management. Rook uses the facilities provided by the underlying cloud-native container management, scheduling and orchestration platform to perform its duties.
```
Comment (parth-gr) [inline README.md:16]: I think we should keep this 16 and 18 line,
This makes me more understandable what Rook can do,
Btw I used this line on my college internship project report XD

Comment (parth-gr) [reply README.md:16]: ohh In see 16 the is preserved,

But 18 can be also there with little modifications
```Rook provides a seamless experience for scheduling, lifecycle management, resource management, security, monitoring, and user experience.```

Comment (travisn) [reply README.md:16]: There are two sentences (lines 18-19) that describe how rook manages Ceph. I guess the previous line 18 seems redundant to me after those other two sentences.

File: Documentation/README.md [hunk 3]
```diff
@@ -4,13 +4,16 @@ title: Rook
...
+Ceph storage to natively integrate with cloud-native environments.
 
-Rook turns storage software into self-managing, self-scaling, and self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management. Rook uses the facilities provided by the underlying cloud-native container management, scheduling and orchestration platform to perform its duties.
+[Ceph](https://ceph.com/) is a distributed storage system that provides file, block and object storage and is deployed in large scale production clusters.
 
-Rook integrates deeply into cloud native environments leveraging extension points and providing a seamless experience for scheduling, lifecycle management, resource management, security, monitoring, and user experience.
+Rook automates deployment and management of Ceph to provide self-managing, self-scaling, and self-healing storage services.
+The Rook operator does this by building on Kubernetes resources to deploy, configure, provision, scale, upgrade, monitor, and monitor Ceph.
```
Comment (jbw976) [inline Documentation/README.md:13]: monitor is mentioned 2x at the end of this sentence :)

```suggestion
The Rook operator does this by building on Kubernetes resources to deploy, configure, provision, scale, upgrade, and monitor Ceph.
```

**Review by jbw976 — APPROVED** [review]
I think this is a reasonable step to take that reflects the current reality of the project interest and investments.  Thank you for taking this step in a conscious way @travisn!

One small grammar suggestion to include, otherwise LGTM! 💪

**Pull request merged**